### PR TITLE
Update sysdeps for xenial

### PR DESCRIPTION
**Pull request by bac**

### CODE REVIEW

**Comment by bac** [comment]
On xenial there is a nodejs in universe that is newer than the one in our Yellow PPA.  We really want the one from the PPA, so the version must be specified. This will need to be updated if we ever update the one in the PPA.

QA:

On [trusty, wily, xenial]:
```
sudo apt install make
make sysdeps
make fast-babel
make gui
make test
```

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4960/
Test PASSed.

File: Makefile [hunk 1]
```diff
@@ -50,6 +50,10 @@ STATIC_CSS_FILES = \
 
 LSB_RELEASE = $(shell lsb_release -cs)
 
+# The nodejs must be versioned in order to get the correct one from the PPA.
+# libfontconfig1 is required by phantom.
+SYSDEPS = nodejs=0.10.37-1chl1~utopic1 python-virtualenv g++ inotify-tools libfontconfig1
```
Comment (hatched) [inline Makefile]: Does this work for Trusty?

Comment (bac) [inline Makefile]: yes.

**Comment by jrwren** [comment]
QA: 
started on a system which already had nodejs from universe installed and got this:
```
The following packages will be REMOVED:
  nodejs-dev npm
The following packages will be DOWNGRADED:
  nodejs
0 upgraded, 0 newly installed, 1 downgraded, 2 to remove and 11 not upgraded.
E: Packages were downgraded and -y was used without --allow-downgrades.
```
I'm OK with this and feel that a manual intervention is GOOD in this non-default environment.

QA instructions were missing a needed clean-all for me 🐣 

I can't seem to build for some reason, but I do not think it is related to this change. I have other issues.
I get http://pastebin.ubuntu.com/16208777/ and http://paste.ubuntu.com/16208621/ and http://pastebin.ubuntu.com/16208604/ 

👍 from me, but I didn't do a full QA.

**Comment by frankban** [comment]
:+1: QA ok on xenial.

**Comment by bac** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**